src/services/audius-api-client/AudiusAPIClient.ts

```diff
@@ -712,7 +712,18 @@ class AudiusAPIClient {
       args,
       true
     )
-    if (!trackResponse) return null
+    // Try the old route method, ensuring that the track once found has the same owner handle
+    if (!trackResponse) {
+      const matches = args.slug.match(/[0-9]{6,}$/)
```

Comment:
worthy a try catch for this?

Reply:
Does `match` throw?

Reply:
meant the whole section. i dont think anything could throw here because parseInt returns NaN on failure. just was thinking aloud

Reply:
(the regex shouldn't match any non-int, unless someone uses a number larger than an int can store I guess)